docs/sdks/python/guides/quickstart.mdx

```diff
@@ -3,4 +3,95 @@ title: Python SDK Quickstart
 description: This quickstart guide will help you get started with the Pieces for Developers Python SDK.
 ---
 
-[//]: # (TODO: Need to write this page)
+## Introduction
+
+The Pieces SDK is a powerful code engine package designed for writing applications on top of Pieces OS. It facilitates communication with a locally hosted server to enable features such as copilot chats, asset saving, and more.
+
+## Installation
+
+To get started with the Pieces SDK, follow these steps:
+
+1. **Download Pieces OS**: Pieces OS serves as the primary backend service, providing essential functionality for the SDK. Download the appropriate version for your operating system:
+   - [macOS](https://docs.pieces.app/installation-getting-started/macos) 
```

Comment:
Should ensure that all the links we use do not have `https://docs.pieces.app` in them so they work on preview links and localhost as well 👍 

I would use absolute paths here: `[macOS](/installation-getting-started/macos)`

Reply:
This goes for the other install links as well